src/Filter/types/index.ts

```diff
@@ -415,6 +416,14 @@ export interface MultiSelectFilterProps {
  */
  export interface MultiSelectFilterOptions extends Omit<Partial<FilterModule<string[]>>, 'description' | 'label'> {}
 
+/**
+ * `ParseMultiValue` is the structure of the option to allow parsing multiple values from a delimited list.
+ */
+export interface ParseMultiValue {
+    enabled: boolean;
```

Comment:
can we have the `enabled` property be optional and default to `false`? (the thinking is that you'll need to "turn on" this special behavior)

Reply:
After some thought, I can get rid of this property altogether and just make parsing dependent on if a delimiter is specified (see other comment regarding delimiter).

Reply:
even better

Reply:
done